scripts/pre_commit_linter.py

```diff
@@ -83,8 +98,20 @@ def _is_javascript_file(filename):
     return filename.endswith('.js')
 
 
+def _is_python_file(filename):
+    """Check if the input filename represents a Python file.
+
+    Args:
+    - filename: str. The name of the file to be checked.
+
+    Returns:
+      bool: True if the filename ends in ".py", and false otherwise
+    """
+    return filename.endswith('.py')
```

Comment:
I was wondering if it might be better if we reduce it to a lambda function so that it is more compact for the filter call below. 
```python
lambda filename : filename.endswith('.py')
```